CHANGELOG.unreleased.md

```diff
@@ -7,6 +7,8 @@
 
 > Users must be able to say: “Nice enhancement, I'm eager to test it”
 
+- [Home] Remove 'tags' filter from the filter selector (PR [#5121](https://github.com/vatesfr/xen-orchestra/pull/5121))
```

Comment:
"...since tags have their own selector"
Otherwise it's hard to understand why we're removing it.